SETUP.md

```
@@ -0,0 +1,242 @@
+
+```bash
+goose session start
+```
+
+---
+
+## Example Troubleshooting Session
+
+Here's an example workflow for analyzing a cluster issue:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)